src/js/__tests__/checkDocumentCSPHeaders-test.js

```diff
@@ -10,10 +10,12 @@
 import {jest} from '@jest/globals';
 import {checkDocumentCSPHeaders} from '../content/checkDocumentCSPHeaders';
 import {ORIGIN_TYPE} from '../config';
+import {setCurrentOrigin} from '../content/updateCurrentState';
 
 describe('checkDocumentCSPHeaders', () => {
   beforeEach(() => {
     window.chrome.runtime.sendMessage = jest.fn(() => {});
+    setCurrentOrigin('FACEBOOK');
```

Comment:
Should we iterate each test with all of the Origins?

Probably not, better to exercise that where the behavior actually changes as a result of that.

Reply:
Agree probably not. I wrote a longer explanation below, but as it currently stands, the code path that uses this value is mocked further down the chain anyway. If that changes, though, we would probably want this coverage.